core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.spec.ts

```diff
@@ -310,6 +310,40 @@ describe('Translation Suggestion Review Modal Component', function() {
         'suggestion_1', 'suggestion_2']);
     });
 
+    it('should accept a suggestion noting that it was edited in its review ' +
+      'message when clicking on accept and review next suggestion button ' +
+      'after updating the suggestion without providing a review message',
```

Comment:
The test name is a bit convoluted. Suggest simplifying to something like "should set suggestion review message to auto-generated note when suggestion is accepted with edits and no user-supplied review message"

Reply:
@sagangwee Done